src/internal/observable/generate.ts

```diff
@@ -47,27 +47,12 @@ export interface GenerateOptions<T, S> extends GenerateBaseOptions<S> {
 }
 
 /**
- * Generates an observable sequence by running a state-driven loop
- * producing the sequence's elements, using the specified scheduler
- * to send out observer messages.
- *
- * ![](generate.png)
- *
- * @example <caption>Produces sequence of 0, 1, 2, ... 9, then completes.</caption>
- * const res = generate(0, x => x < 10, x => x + 1, x => x);
- *
- * @example <caption>Using asap scheduler, produces sequence of 2, 3, 5, then completes.</caption>
- * const res = generate(1, x => x < 5, x => x * 2, x => x + 1, asap);
- *
- * @see {@link from}
- * @see {@link Observable}
- *
  * @param {S} initialState Initial state.
```

Comment:
We can actually remove all the type information like `{S}` as well. We're gleaning that stuff from TypeScript now.

Reply:
remove just the type part of the overload docblocks? (e.g. `{S}`)
or remove all overload docblocks (e.g. `@param {S} initialState Initial state.`)?

Reply:
just the {s}, sorry for the super late reply